custom_components/eventsensor/sensor.py

```diff
@@ -170,6 +178,7 @@ def async_update_sensor(event: Event):
                 }
                 _LOGGER.debug("%s: New state: %s", self.entity_id, self._state)
                 self.async_write_ha_state()
+                asyncio.ensure_future(self.async_set_idle())
```

Comment:
@koying,

In addition to the general comment about this PR, the implementation is also wrong... sorry :(

What happens if > 1 events are fired in a short period of time? :)

Reply:
I should cancel the future, right?
I'm really not a python guru ;)

Reply:
yes, is not a good thing to fire those without control.

There are already ways to do that kind of things in HA Core, and it is better to use them. For example, in `homeassistant.helpers.event` there are some helpers to trigger some action after some time. You could explore their usages in the HA Core code, for inspiration or just to copycat how to do it